api-messaging.md

```diff
@@ -0,0 +1,98 @@
+# Messaging API
+
+## Version
+
+- v1.0: Initial draft (2021-11-25)
+
+## General
+
+### Encoding
+
+UTF-8 encoding shall be used at all times.
+
+### Staging base URL
+
+https://api-staging.safereach.com/blaulicht
+
+### Live base URL
+
+https://api.safereach.com/blaulicht
+
+### Usage
+
+The API is a simple transactional messaging API.
+
+API requests are limited by the Fair Use Policy and configured account limits.
+
+### Authentication
+
+For API usage your `customerId`, an automatic alarm trigger user `username` and `password` are needed.
+
+### Models
+
+#### MessageTarget
+
+- channel[]: List of `SMS`, `EMAIL`, `VOICE`, `SAFEREACH`
+- msisdn: target phone number (mandatory for `SMS` and `VOICE`)
+- email: target email address (mandatory for `EMAIL`)
+
+> `SAFEREACH` will automatically determine the best message channel which will be the app in most cases
+
+**Example:**
```

Comment:
What if more than one channel is defined? How is the best channel determined? Is there any ranking process? Are we sending it out to all channels?

Reply:
Send to all correctly defined channels. If all channels are not reachable, send error response with `msisdn` or `email` and error message.